contracts/proxy/ImplementationReference.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
can we make all contracts that are Ownable also Claimable? Adding a claimOwnership() function helps prevent mistakes when transferring contract ownership.

Reply:
Good call, we'll add a task for that